tasks/main.yml

```diff
@@ -16,11 +16,11 @@
 - name: Elasticsearch Plugins
   include: plugins.yml
 
-- name: Install elasticsearch plugin
-  shell: >
-    {{ elasticsearch_plugin_cmd }} install -b {{ item }}
-  when: item not in plugins.stdout_lines
-  with_items: "{{ elasticsearch_plugins }}"
+# - name: Install elasticsearch plugin
+#   shell: >
+#     {{ elasticsearch_plugin_cmd }} install -b {{ item }}
+#   when: item not in plugins.stdout_lines
+#   with_items: "{{ elasticsearch_plugins }}"
```

Comment:
can this be deleted rather than commented out please

Reply:
This has been deleted.